Applications/SlicerApp/Testing/Cpp/qSlicerModuleFactoryManagerTest1.cxx

```diff
@@ -29,7 +29,8 @@
 // VTK includes
 #include <vtkNew.h>
 
-int qSlicerModuleFactoryManagerTest1(int argc, char * argv[])
+int
```

Comment:
Keep as single line. Why woud you break into separate lines?

Reply:
done.